mobile/src/main.jsx

```diff
@@ -37,7 +37,7 @@ const renderAppWithPersistedState = persistedState => {
           cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
           initBar()
         } else {
-          onError(store.dispatch, store.getState)()
+          onError(store.dispatch, store.getState)({ message: 'Client has been revoked' })
```

Comment:
maybe you can use https://github.com/cozy/cozy-files-v3/pull/207/files#diff-f96699990a60f265f8d3b7bd25d3e9a6R8 ?